### DISCUSSION TITLE
Facebook Private Group Posts Scraper

### DISCUSSION THREAD

**Discussion by mnisbet** [kevinzg/facebook-scraper#160] [Ideas]
Hello,

Can this also scrape group posts of private groups that I am a member of? If not, would it be simple enough for someone to adapt the code to achieve this?

Thank you for your help.

Mike

**Comment by kevinzg** [comment]
I think the current authentication method (user and password) has some issues. The plan was to add cookie based authentication, i.e. you login on the browser and then use those cookies to do the requests on the scraper. This hasn't been implemented, but it shouldn't be too hard.

Regarding groups, they also have some issues, but I haven't look into them.

**Comment by goodcoderfromeurope** [comment]
hi, has anybody solved this problem? any other opensource app for this?
